test/rubygems/test_gem_commands_contents_command.rb

```diff
@@ -105,6 +105,22 @@ def test_execute_missing_single
     assert_empty @ui.error
   end
 
+  def test_execute_missing_version
```

Comment:
After having revised the PR this test isn't strictly necessary to the functionality I'm updating, but I have left it in anyway because it seems to be a missing case. Can remove if it feels out of scope though.

Reply:
I'm happy with including that test, but I would split it into a separate commit to make it clear that it's not related to the bug fix.

Reply:
Done.